include/GoogleSync/GoogleSync.php

```diff
@@ -146,7 +175,7 @@ public function addUser($id, $name)
     public function delUser($id)
```

Comment:
I wrote that method very early on as the mirror method to addUser. I can't think of any current use for it. Should I get rid of it?

Reply:
Are any chance if it will be needed? If it works as expected and tested and also you think it would be usefull in a possible future request later so it's nessessary, you can keep it, but I don't see any reason. Next time please don't implement unnecessary fetures, just think about what if it has a bug somewhere what we can not realise just because there is no any use case and we can not try it before the end-users or custom developers does.. (but it's only my view, it's up to you :) )